rust/build.rs

```diff
@@ -0,0 +1,26 @@
+// See: https://rust-lang.github.io/rust-bindgen/tutorial-3.html
+use std::{env, path::PathBuf};
+
+const WRAPPER: &str = "lib/wrapper.h";
+
+fn main () {
+  println!("cargo:rerun-if-changed={}", WRAPPER);
+  println!("cargo:rustc-link-lib={}={}", "dylib", "xenctrl");
```

Comment:
why not linking against xenstore as well?

Reply:
Because I forgot that. :slightly_smiling_face: